python-sdk/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pachyderm_sdk"
-version = "0.0.1"
+version = "0.0.0+dev"
```

Comment:
Is there a way we could set this to the incoming version? ($CIRCLE_TAG I believe?)

Reply:
The version gets set with the release tag when building and publishing